pom.xml

```diff
@@ -183,6 +223,36 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <configuration>
+                            <attach>true</attach>
+                            <excludePackageNames>
+                                com.zaxxer.nuprocess.internal,
+                                com.zaxxer.nuprocess.linux,
+                                com.zaxxer.nuprocess.osx,
+                                com.zaxxer.nuprocess.windows
+                            </excludePackageNames>
+                            <show>public</show>
+                            <sourceFileExcludes>
+                                <exclude>**/NuProcessFactory.java</exclude>
+                            </sourceFileExcludes>
+                        </configuration>
+                    </plugin>
+
+                    <!-- nexus-staging-maven-plugin replaces the standard maven-deploy-plugin for releases and
+                         is useful for ensuring artifacts are all-or-nothing, as well as allowing artifacts to
+                         be reviewed before they're made public -->
+                    <plugin>
+                        <groupId>org.sonatype.plugins</groupId>
+                        <artifactId>nexus-staging-maven-plugin</artifactId>
+                        <extensions>true</extensions>
+                        <configuration>
+                            <nexusUrl>https://oss.sonatype.org/</nexusUrl>
+                            <serverId>ossrh</serverId>
```

Comment:
I have not added `<autoReleaseAfterClose>true</autoReleaseAfterClose>` here, which means after performing a release you have to either login to https://oss.sonatype.org and release the artifacts or use `mvn nexus-staging:release`. Not using auto-release was useful in testing my release setup, but it may be desirable to set it here going forward.